src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,167 @@
+package wolox.training.models;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Book {
+
+    private String genre;
+
+    @NotNull
+    @Column(nullable = false)
+    private String author;
+
+    @NotNull
+    @Column(nullable = false)
+    private String image;
+
+    @NotNull
+    @Column(nullable = false)
+    private String title;
+
+    @NotNull
+    @Column(nullable = false)
+    private String subtitle;
+
+    @NotNull
+    @Column(nullable = false)
+    private String publisher;
+
+    @NotNull
+    @Column(nullable = false)
+    private String year;
+
+    @NotNull
+    @Column(nullable = false)
+    private String pages;
+
+    @NotNull
+    @Column(nullable = false, unique = true)
+    private String isbn;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long id;
```

Comment:
The only thing missing here is adding a get for this attribute. The set is not necessary since the id are generated automatically by the db.

Reply:
My mistake, I'll add it